assets/node-exporter/daemonset.yaml

```
@@ -182,6 +185,12 @@ spec:
       - name: node-exporter-kube-rbac-proxy-config
         secret:
           secretName: node-exporter-kube-rbac-proxy-config
+      - configMap:
+          items:
+          - key: accelerators-collector-config.yaml
+            path: accelerators-collector-config.yaml
```

```suggestion
          - key: config.yaml
            path: config.yaml
```
(given the change, to the folder name's, above)

---

fixed